### PR TITLE
Add itemTree render cell option

### PR DESCRIPTION
**Pull request by windingwind**

### CODE REVIEW

File: chrome/content/zotero/xpcom/itemTreeManager.js [hunk 1]
```diff
@@ -88,6 +88,17 @@ class ItemTreeManager {
 	 *         // return: the data to display in the column
 	 *         return item.getField('title').split('').reverse().join('');
 	 *     },
+	 *     cellRenderer: (index, data, column) => {
```
Comment (adomasven) [inline chrome/content/zotero/xpcom/itemTreeManager.js:91]: Signature `renderCell: (index, data, column)`

**Comment by windingwind** [comment]
@dstillman This PR is ready to be merged if no further changes are required

**Comment by RayKr** [comment]
手动 @windingwind 风大查看，被这个列bug折磨够了啊，求快快merge后修复🤣

**Comment by dstillman** [comment]
Available in the latest beta

**Pull request merged**